src/main/java/org/elasticsearch/search/suggest/SuggestParseElement.java

```diff
@@ -44,12 +47,15 @@ public SuggestParseElement(Suggesters suggesters) {
 
     @Override
     public void parse(XContentParser parser, SearchContext context) throws Exception {
-        SuggestionSearchContext suggestionSearchContext = parseInternal(parser, context.mapperService(), context.shardTarget().index(), context.shardTarget().shardId());
+        SuggestionSearchContext suggestionSearchContext = parseInternal(parser, context.mapperService(), context.shardTarget().index(), context.shardTarget().shardId(), context, context);
         context.suggest(suggestionSearchContext);
```

Comment:
no need to anymore, we can remove the whole suggest stuff due to https://github.com/elastic/elasticsearch/commit/7efc43db25495b92833c4bc804f604b82597c157